src/visGeometry/VisAgent.ts

```diff
@@ -65,9 +68,9 @@ export default class VisAgent {
         };
     }
 
-    public setColor(color: Color, colorIndex: number): void {
-        this.color = color;
-        this.colorIndex = colorIndex;
+    public setColor(colorInfo: { color: Color; colorId: number }): void {
```

Comment:
what is this change doing? Is this an object that you pass around in other places?  I will read on...

Reply:
I made this change so that I could call one function in the `colorHandler` that returns both things needed for this function. Before calls to `setColor` were calling two functions that each returned one of the two arguments. I know I could have spread out the return into arguments so I didn't need to change this function, but I think that looks more confusing